lib/structures/User.js

```diff
@@ -5,7 +5,24 @@ const Endpoints = require("../rest/Endpoints");
 
 /**
 * Represents a user
-* @extends Base
+* @prop {Number?} accentColor The user's banner color, or null if no banner color (REST only)
+* @prop {String?} avatar The hash of the user's avatar, or null if no avatar
+* @prop {String} avatarURL The URL of the user's avatar which can be either a JPG or GIF
+* @prop {Object?} avatarDecorationData The data of the user's avatar decoration, or null if there is no avatar decoration
+* @prop {String?} avatarDecorationURL The URL of the user's avatar decoration which will be a PNG
+* @prop {String?} banner The hash of the user's banner, or null if no banner (REST only)
+* @prop {String?} bannerURL The URL of the user's banner
+* @prop {Boolean} bot Whether the user is an OAuth bot or not
+* @prop {Number} createdAt Timestamp of the user's creation
+* @prop {String} defaultAvatar The hash for the default avatar of a user if there is no avatar set
+* @prop {String} defaultAvatarURL The URL of the user's default avatar
+* @prop {String} discriminator The discriminator of the user
+* @prop {String} id The ID of the user
+* @prop {String} mention A string that mentions the user
+* @prop {Number?} publicFlags Publicly visible flags for this user
+* @prop {String} staticAvatarURL The URL of the user's avatar (always a JPG)
+* @prop {Boolean} system Whether the user is an official Discord system user (e.g. urgent messages)
+* @prop {String} username The username of the user
```

Comment:
`@prop` declarations are not to be used – they were removed in #113 (the suggestion below reverts the change)

```suggestion
* @extends Base
```